package.json

```diff
@@ -18,7 +18,7 @@
     "lint-js": "eslint --ext=.js,.jsx gulpfile.js server/ common/ config/ client/",
     "lint-json": "npm run lint-server && npm run lint-challenges && npm run lint-resources && npm run lint-utils",
     "only-once": "npm run create-rev && echo '/****/' && echo 'Seeding Database' && echo '/****/' && node seed && echo '/****/' && echo 'Seeding Completed' && echo '/****/'",
-    "prelint-js": "npm run create-rev",
+    "prelint-js": "npm run create-rev && gulp generate-migration-map",
```

Comment:
Can you make this more friendly, to our current contributors? This will most certainly crash setups for contributors?

Reply:
How so? `gulp` is no longer a global dependency

Reply:
Well for one, it will not work when people are doing a fresh clone and `npm run only-once`

Reply:
I think you would have to 

```
"only-once": "npm run create-rev && gulp generate-migration-map && ...
```